src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,164 @@
+  caption="Credit: diagram of the Milgram experiment from Wikipedia"
+/>
+
+In reality, there were no real shocks being administered, but this was unknown to the study participant playing the role of the Teacher. The actor playing the role of the Learner was instructed to pretend to be in pain, and recordings of increasingly alarming cries of pain were played when shocks were administered.
+
+When the highest shock level was reached, the learner fell completely silent.
+
+During the experiment, if the teacher showed hesitation to continue the experiment, the experimenter gave them a series of specific instructions to continue:
+
+1. Please go on.
+1. The experiment requires that you continue.
+1. It is _absolutely essential_ that you continue.
+1. You have no other choice; you _must_ go on.
+
+These "prods" were given in order, and the second was only given if the teacher refused to continue after the first (and so on).
+
+The experiment was halted when the teacher refused to continue despite all four prods, or when the teacher reached the highest shock level three times in a row.
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-photos"
+  alt="Photos from the original Milgram experiment, credit Stanley Milgram."
+  caption="Photos from the original Milgram experiment, credit Stanley Milgram"
+/>
+
+### What would you do?
+
+Let's take a minute to pause here - put yourself in the shoes of the teacher. You're in a lab, and you're told that you're taking part in a study on learning and memory. Everything you're seeing and hearing seems to be controlled, official, and legititmate.
+
+Do you think you would continue to administer shocks to the learner, even when they were clearly in pain?
+
+What do you think the average person would do?
+
+<br />
+<Colophon />
+
+### The Experiment: Results
+
+As you might expect, every participant in the study indicated a desire to stop the experiment at least once.
+
+Before conducting the experiment, researchers surveyed a cohort of students and fellow researchers, to get a sense of what they thought the average person would do. The average guess was that 1.2% of people would continue the experiment through the full 450V shock.
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/predicted-results"
+  alt="Predicted Results: 1.8% complete the full series of shocks, 98.2% refuse to continue at some point"
+  caption="Surveys predicted that very few participants would carry out the full series of shocks"
+/>
+
+<br />
+
+However, the actual results of the experiment revealed a surprising truth about human behavior:
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/actual-results"
+  alt="Actual Results: 59.1% completed the full series of shocks, 40.9% refused to continue at some point"
+  caption="The experimental results were much more grim."
+/>
+
```

Comment:
But, that study was in 1961. We've evolved culturally since then, I don't think this could happen now? 

Nope, replicated in 2009, we're still awful: https://www.apa.org/news/press/releases/2008/12/replicating-milgram